demisto_sdk/commands/common/files/ini_file.py

```diff
@@ -16,7 +17,14 @@ def is_model_type_by_path(cls, path: Path) -> bool:
 
     def load(self, file_content: bytes) -> ConfigParser:
         config_parser = ConfigParser(allow_no_value=True)
-        config_parser.read_string(super().load(file_content))
+        file_content_as_text = super().load(file_content)
```

Comment:
what happens if this fails?

Reply:
will raise `FileLoadError` which is what we want, it means we couldn't load the object into the requested object.